Mathlib/RingTheory/Trace/Defs.lean

```diff
@@ -152,7 +155,8 @@ section TraceForm
 variable (R S)
 
 /-- The `traceForm` maps `x y : S` to the trace of `x * y`.
-It is a symmetric bilinear form and is nondegenerate if the extension is separable. -/
+It is a symmetric bilinear form and is nondegenerate if the extension is separable.-/
+@[stacks 0BIK "Trace pairing"]
```

Comment:
I'm confused by the purpose of the text after `@[stacks]`. Sometimes it is used to describe the difference from the stacks result and ours but I'm not sure what it's for here.

Reply:
I think we should use it to describe the difference. But there are many kinds of differences. The difference in naming, in content (the mathlib theorem is only a part of theorem in stacks project) and in hypothesis.